#! /bin/sh /usr/share/dpatch/dpatch-run
## DP: fix assembly location

@DPATCH@

--- mono-zeroconf/src/mono-zeroconf.pc.in.old	2007-12-20 05:04:09.000000000 +0100
+++ mono-zeroconf/src/mono-zeroconf.pc.in	2007-12-20 05:04:19.000000000 +0100
@@ -6,5 +6,5 @@
 Description: Mono library for universal cross platform Zeroconf support via a unified API with built in support for mdnsd and optional support for Avahi.
 Version: @VERSION@
 Requires:
-Libs: -r:${libdir}/mono/mono-zeroconf/Mono.Zeroconf.dll
+Libs: -r:${libdir}/cli/Mono.Zeroconf-1.0/Mono.Zeroconf.dll
 
